### DISCUSSION TITLE
Consistent backup of data directory without stopping the server

### DISCUSSION THREAD

**Discussion by baurmatt** [Koenkk/zigbee2mqtt#9082] [General]
Hey,

I was wondering if a simple `cp`/`rsync`/... of the `data` directory gives me a consistent backup. Or do I have to stop zigbee2mqtt for this?

Thanks!

Regards,
Matthias

**Comment by Koenkk** [comment]
This will give you a consistent backup, a backup is made every 24 hours and when stopped.

**Reply by baurmatt** [reply]
Sorry, I can't completely follow. What gives me a consistent backup? To which location is this backup made every 24h? And how do I know the time when this backup is made? Is there an API to trigger the backup?

**Reply by Koenkk** [reply]
There is no API to trigger the backup. Making a backup of the data directory every 24 hours should give you a consistent backup.
